docs/SPEC.md

```diff
@@ -2571,23 +2571,80 @@ from(bucket: "telegraf/autogen")
 
 #### Difference
 
-Difference computes the difference between subsequent non null records.
+Difference computes the difference between subsequent records.  
+Every user-specified column of numeric type is subtracted while others are kept intact.
 
 Difference has the following properties:
 
 * `nonNegative` bool
     nonNegative indicates if the derivative is allowed to be negative.
```

Comment:
*nit* This was here already but it should say `...if the difference is allowed to be...`

Reply:
Sorry, I can't get what you are saying 😅

Reply:
There was a copy/paste error, it says "derivative" but it should say "difference".

Reply:
Oh! Even reading it multiple times I couldn't see it! Thank you